### ISSUE TITLE
Introduce spectatorcolour

### ISSUE THREAD

**Issue by TheAssassin** [blue-nebula/base#190]
Right now, spectating people are assigned the same color in server browsers and the in-game chat as people in "team neutral". This color is managed in the variable `/teamneutralcolour`.

A very annoying disadvantage of this approach is that, from the server browser, you cannot differentiate between people who actively play, and people who idle or spectate otherwise (in `ffa` modes at least). To fix that, we should introduce a new variable and assign those a different color than for the "neutral team".

Protocol wise it should work in a backwards compatible manner. The team colors are sent for every player by the server anyway, at least on the "info port".

**Comment by robalni** [comment]
Yes, this is something I want. It has happened many times that I have clicked on a server to play ffa with someone and then noticed that the person was spectating.

**Comment by MoonPadUSer** [comment]
Yeah, that seems like a neat thing to have.
